ci-operator/jobs/stackrox/scanner/stackrox-scanner-release-2.37-postsubmits.yaml

```diff
@@ -0,0 +1,106 @@
+postsubmits:
+  stackrox/scanner:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-2\.37$
+    cluster: build05
```

Comment:
Curious: Is this cluster name generated? or something you picked? (put another why, why 05 vs. a different one)

Reply:
everything under `ci-operator/jobs` is generated